lib/util/errors.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const { DiagnosticSeverity } = require("@stoplight/types");
```

Comment:
Suggest not bringing this package in. Just set `severity` to a hardcoded val equal to that enum.

Reply:
Well, I guess we could only expose:
- `path` - json path pointing at some value in the document that caused an error)
- `source` -  uri of document.
- `code` / `message`